File: src/codes/Users_Manual/sampling_config/modulation_0.cs
~using AUTD3Sharp;
~using AUTD3Sharp.Modulation;
~using static AUTD3Sharp.Units;
~{
var m = new Sine(150u * Hz).WithSamplingConfig(4000 * Hz);
~}
// or
~{
var m = new Sine(150u * Hz).WithSamplingConfig(TimeSpan.FromMicroseconds(250));
~}

File: src/codes/Users_Manual/sampling_config/modulation_0.rs
# use autd3::prelude::*;
# #[allow(unused_variables)]
# fn main() {
let m = autd3::modulation::Sine::new(150 * Hz)
            .with_sampling_config(4000 * Hz);
// or
let m = autd3::modulation::Sine::new(150 * Hz)
            .with_sampling_config(std::time::Duration::from_micros(250));
# }